spec/views/recipes/show.html.erb_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe "recipes/show", type: :view do
+  before(:each) do
+    @recipe = assign(:recipe, Recipe.create!())
+  end
+
+  it "renders attributes in <p>" do
+    render
+  end
+end
```

Comment:
if there's no show.html.erb files for recipes...is this file needed?

Reply:
I think we just missed deleting all these tbh. We dont need them, they were just auto created when we did our scaffolding